bin/test_flakiness

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env ruby
+
+# Determine how flaky our UI tests by measuring recent failure rate
+# using data from the Saucelabs API. Prints out every UI test from
+# least flaky to most flaky.
+
+require_relative '../deployment'
+require 'rest-client'
+
+username = ENV['SAUCE_USERNAME'] || CDO.saucelabs_username
+key = ENV['SAUCE_ACCESS_KEY'] || CDO.saucelabs_authkey
+
+PER_REQUEST = 1500 # maximum returned per API call (undocumented)
```

Comment:
I'd like to be able to override these numbers at the command line, in case I want more history